updater/lib/dependabot/updater/group_update_creation.rb

```diff
@@ -57,11 +57,14 @@ def compile_all_dependency_changes_for(group)
 
         notices = []
```

Comment:
- [ ]

Reply:
Sorry, I couldn't understand what you mean by that.